drf/university/serializers.py

```diff
@@ -1,10 +1,24 @@
 from django_countries.serializers import CountryFieldMixin
 from rest_framework import serializers
 from .models import University
+from school.models import School
 
 
 class UniversitySerializer(CountryFieldMixin, serializers.ModelSerializer):
+    schools = serializers.HyperlinkedRelatedField(
+        many=True, view_name='school-detail', queryset=School.objects.all())
```

Comment:
Same thing. This is inefficient because for every school you will execute query to get all universities. Better would be to prepare data in view overriding _get_queryset_ method